cabal-install/Distribution/Client/Sandbox.hs

```diff
@@ -141,28 +138,7 @@ snapshotDirectoryName = "snapshots"
 -- | Non-standard build dir that is used for building add-source deps instead of
 -- "dist". Fixes surprising behaviour in some cases (see issue #1281).
 sandboxBuildDir :: FilePath -> FilePath
-sandboxBuildDir sandboxDir = "dist/dist-sandbox-" ++ showHex sandboxDirHash ""
```

Comment:
@ttuegel the has function is moved from here i.e. not introduced in this patch. Though no idea why this choice in the first place.